extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/vertx/OpenTelemetryHttpServerMetrics.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.opentelemetry.runtime.tracing.vertx;
+
+import java.util.Optional;
+
+import io.vertx.core.Context;
+import io.vertx.core.http.impl.HttpServerRequestInternal;
+import io.vertx.core.spi.metrics.HttpServerMetrics;
+import io.vertx.core.spi.observability.HttpRequest;
+
```

Comment:
Can you add a comment here re: mutual exclusivity with micrometer extension?

Reply:
Sure.